src/EducationService.Mappers/Models/EducationInfoMapper.cs

```diff
@@ -0,0 +1,26 @@
+﻿using LT.DigitalOffice.EducationService.Mappers.Models.Interfaces;
+using LT.DigitalOffice.EducationService.Models.Db;
+using LT.DigitalOffice.EducationService.Models.Dto.Models;
+
+namespace LT.DigitalOffice.EducationService.Mappers.Models;
+
+public class EducationInfoMapper : IEducationInfoMapper
+{
+  public EducationInfo Map(DbUserEducation dbUserEducation)
+  {
+    return dbUserEducation is null
+      ? null
+      : new EducationInfo
+      {
+        Id = dbUserEducation.Id,
+        UserId = dbUserEducation.UserId,
+        UniversityName = dbUserEducation.UniversityName,
+        QualificationName = dbUserEducation.QualificationName,
+        EducationFormId = dbUserEducation.EducationFormId,
```

Comment:
map and put in response Form id and FormName.
Create EducationFormInfo model (Guid Id, string Name)

Reply:
Done